meta/main.yml

```diff
@@ -9,13 +9,13 @@ dependencies:
     version: 448b1cb4daee97c1402bcfe15e4bdda7e8a83dab
   - name: os-hardening
     src: git+https://github.com/UKHomeOffice/ansible-role-os-hardening
-    version: 9d2bacc1a406eaa695e319f0d494f4ab9edaac2b
+    version: abcc5102c948e87ccb7a06e7d10b5227ece0374f
```

Comment:
The evidence is documented in Jira in the following tickets. These Jira tickets were closed in the last sprint:
- CROSSING-7064
- CROSSING-7065
- CROSSING-7067
- CROSSING-7069
-